.gitignore

```diff
@@ -13,3 +13,5 @@ ENV*
 
 
 .coveralls.yml
+
+.DS_Store
```

Comment:
I don't mind it so much, but I think it belongs the global one: `~/.gitignore`. I think you'll get this in other OSS because it's not in the Github's [python template](https://github.com/github/gitignore/blob/master/Python.gitignore).